app/controllers/searches_controller.rb

```diff
@@ -14,7 +14,7 @@ def results
     @title = 'Search'
     @tagnames = params[:id].split(',')
     @users = SearchService.new.find_users(params[:id], limit = 10)
-    @nodes = SearchService.new.find_nodes(params[:id], 100, params[:order].to_s.to_sym, params[:type].to_s.to_sym)
+    @nodes = SearchService.new.find_pages(params[:id], 100, params[:order].to_s.to_sym, params[:type].to_s.to_sym)
```

Comment:
Like here, shouldn't we search both notes and pages? We could combine two "find" calls, or does this mean we may still need a third "find_nodes"?

Reply:
hi, Jeff, all of this concerns are fixed on the current branch that we are working and this was something that I mentioned at the time Camila opened this PR... because it makes no sense to separate things like it was implemented.

Reply:
Hi, I'm sorry, it's a combination of issues on my end -- it's a big PR, doing multiple things, so it's a little hard to keep it all in my head at once when providing feedback. Maybe in the future, breaking ones like this into multiple parts (say, limits, distincts, and separation of node/notes/pages?) could help.  But also, I didn't understand originally that the central premise of the PR was to present separate results in the Typeahead for NOTES vs. PAGES, as in this display there are only NOTES:

![image](https://user-images.githubusercontent.com/24359/45395922-742d8780-b606-11e8-9c2c-1d87e1b783f5.png)

Now I understand, and I think separate sections for NOTES and PAGES sounds wonderful. However, I still want to be sure that, for example, a call like:

`SearchService.textSearch_all()` 

will return both notes and pages. They could be together, merged, or concatenated separately, but we should be sure they aren't forgotten -- i'm mainly thinking if there are any cases where someone would search and expect to see a page, but somehow we've forgotten to display that type.